2025-03-10 15:44:02,199 - nisto - INFO - Starting Network Topology Designer
2025-03-10 15:44:02,669 - nisto - ERROR - Unhandled exception in main
Traceback (most recent call last):
  File "/home/cam/Desktop/NISTO/network-topology-designer/src/main.py", line 63, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/cam/Desktop/NISTO/network-topology-designer/src/main.py", line 54, in main
    main_window = MainWindow()
                  ^^^^^^^^^^^^
  File "/home/cam/Desktop/NISTO/network-topology-designer/src/controllers/main_window.py", line 43, in __init__
    self.mode_manager = ModeManager(
                        ^^^^^^^^^^^^
  File "/home/cam/Desktop/NISTO/network-topology-designer/src/controllers/mode_manager.py", line 35, in __init__
    self.view.viewport().installEventFilter(self)
TypeError: installEventFilter(self, a0: Optional[QObject]): argument 1 has unexpected type 'ModeManager'
